src/components/20-molecules/datepicker/index.js

```diff
@@ -662,7 +648,21 @@ class AXADatepicker extends NoShadowDOM {
     onBlur(e);
   }
 
-  handleButtonOkClick() {
+  fireEvents(validDate) {
+    if (validDate) {
+      const { name } = this;
+      const value = this.formatDate(validDate);
+      const details = { value, date: validDate, name };
+      fireCustomEvent('axa-change', value, this, { bubbles: false });
+      fireCustomEvent('change', details, this, { bubbles: false });
+    }
+  }
+
+  handleDatepickerCalendarCellClick(e) {
+    e.preventDefault();
+    e.stopPropagation();
+    e.target.blur(); // prevent the ugly focus ring after the click
+
```

Comment:
Why did you moved that whole code block? That makes comparing the diff real hard.

Reply:
initdate needs to be called after the 'outputdate' property has been filled, because this property indicates that the user interacted with the component already.